### PR TITLE
Update vendor containers/common

### PR DESCRIPTION
**Pull request by dfr**
This fixes a FreeBSD regression which prevented creating new containers.

[NO NEW TESTS NEEDED]

#### Does this PR introduce a user-facing change?

```release-note
None
```

### CODE REVIEW

**Review by vrothberg — APPROVED** [review]
LGTM, thank you!

File: go.mod [hunk 1]
```diff
@@ -13,13 +13,13 @@ require (
 	github.com/containernetworking/cni v1.1.2
 	github.com/containernetworking/plugins v1.3.0
 	github.com/containers/buildah v1.31.1-0.20230710135949-9c9a344b9874
-	github.com/containers/common v0.55.1-0.20230713173316-9e5d4a690901
+	github.com/containers/common v0.55.1-0.20230719211449-b94e030a5ea0
```
Comment (TomSweeneyRedHat) [inline go.mod:16]: This should really be v0.56.*, not v0.55.*.   The v0.55 c/common is for Podman v4.6, and is not meant for Podman upstream/v4.7-dev.

Comment (dfr) [reply go.mod:16]: I just used the latest hash from c/common's main branch. There isn't a v0.56 branch in c/common yet so I'm not sure what to do here.

**Comment by dfr** [comment]
@ashley-cui I would like to get something like this into podman's v4.6 branch but I'm not sure what the right way is. Should I merge the specific commit in c/common which fixes my problem into common's v0.55 branch first?

**Comment by dfr** [comment]
> @dfr Yep! best way is to open a PR against c/common's v0.55 branch, backporting the fix to the branch.

Done, thanks!

**Comment by dfr** [comment]
What is the best way forward for this one - @TomSweeneyRedHat suggested using common v0.56 but the branch doesn't exist in c/common yet. I would like to get this (or something like it) merged into main soon - at the moment podman builds from main on freebsd can't create new containers.

File: go.mod (hunk 1 above)
Comment (vrothberg) [reply go.mod:16]: @TomSweeneyRedHat, that's go-mod slang which sometimes guesses the wrong version.  The only source of truth is the short ID after the last dash which points to the specific commit.

**Comment by dfr** [comment]
Rebased

**Comment by dfr** [comment]
Looks like c/common was already updated so I'm closing this PR.

**Pull request closed without merging**